accepted/future-releases/0546-patterns/feature-specification.md

```diff
@@ -144,12 +144,13 @@ matching constructs to make working with that style enjoyable:
 ```dart
 double calculateArea(Shape shape) =>
   switch (shape) {
-    case Square(length: var l) => l * l;
-    case Circle(radius: var r) => math.pi * r * r;
+    Square(length: var l) => l * l,
+    Circle(radius: var r) => math.pi * r * r,
```

Comment:
Nit: Usually in the spec when we're giving examples of lists, map literals, enums, or invocation arguments we don't include the optional final `,`.  I'd prefer for us to be consistent with that practice and leave off the optional final `,` from our examples here.

Reply:
Done.